ui/components/component-library/base-avatar/base-avatar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+
+function BaseAvatar(props) {
```

Comment:
Let's write our components out as named arrow function exports 
```suggestion
export const BaseAvatar = ({ size }) => {
```